config/environments/local/local.node.config.toml

```diff
@@ -80,6 +81,7 @@ WeightBinaries = 1
 WeightSteps = 1
 TxLifetimeCheckTimeout = "10m"
 MaxTxLifetime = "3h"
+MaxTxSizeForL1 = 131072
```

Comment:
this should be 128KB

Reply:
this is 128 KB it's 1024*128 = 131072 bits